src/editor/Collaboration.js

```diff
@@ -12,14 +12,22 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false
         });
+        var hash = location.hash.replace(/^#/, "");
```

Comment:
Can you add a comment above saying that this is only temporary and it needs to be moved to the Bramble API?
Also, can you file a follow up issue in Thimble for this?